mobile/services/mutations/change-password.mutation.ts

```diff
@@ -0,0 +1,17 @@
+import { useMutation } from "@tanstack/react-query";
+import { changePassword, ChangePasswordPayload } from "../definitions.api";
+
+export const useChangePasswordMutation = () => {
+  return useMutation({
+    mutationKey: ["changePassword"],
```

Comment:
please add the key in `pollingStationKeys` in `queries.service.ts` file

Reply:
done